package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "gulp-util": "^3.0.6",
+    "path": "^0.12.7",
```

Comment:
Getting ready to pull this in – is it necessary to require [this alternative `path` module](https://www.npmjs.com/package/path) rather than the [Node.js standard `path` module](https://nodejs.org/docs/latest-v0.10.x/api/path.html#path_path_basename_p_ext)? It looks like it's been stable since 0.10, so I think we may be able to use the default.